cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java

```diff
@@ -960,13 +967,61 @@ public void print(String message, boolean newLine, boolean collect) {
         }
     }
 
+    @Override
+    public void printDMR(ModelNode node) {
+        if (getConfig().isOutputJSON() && getConfig().isColorOutput()) {
+            printLine(node.toJSONString(false), node.get("outcome").asString());
+        } else if (getConfig().isOutputJSON()) {
+            printLine(node.toJSONString(false));
+        } else if (getConfig().isColorOutput()) {
+            printLine(node.toString(), node.get("outcome").asString());
+        } else {
+            printLine(node.toString());
+        }
+    }
+    private String colorizeMessage(String message, String outcome) {
+        if (!"".equals(outcome.toLowerCase())) {
+            switch (outcome) {
+                case "success":
```

Comment:
Perhaps check that Util already expose this constant, in case not, could be interesting to add them.

Reply:
There is one, `return operationResponse.hasDefined(Util.OUTCOME) && operationResponse.get(Util.OUTCOME).asString().equals(Util.SUCCESS);` where `Util.SUCCESS` == "success". Using a method would not allow me to use `switch`, which I preferred in this case.